sklearn/multilabel/classifier_chain.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+from ..base import BaseEstimator, clone
+from ..utils import check_random_state
+from scipy.sparse import issparse
+
+
+class ClassifierChain(BaseEstimator):
+    """
+    A multi-label model that arranges binary classifiers into a chain. Each
```

Comment:
I'm happy to change the wording of this docstring but I'm unclear on the definition of "multi-output".  My implementation follows the original paper which was specifically a multi-label algorithm.